code/modules/vore/eating/vorepanel_vr.dm

```diff
@@ -3263,40 +3265,85 @@ var/global/list/belly_colorable_only_fullscreens = list("a_synth_flesh_mono",
 			host.vore_selected.nutrition_percent = new_new_nutrition
 			. = TRUE
 		if("b_burn_dmg")
-			var/new_damage = tgui_input_number(user, "Choose the amount of burn damage prey will take per tick. Ranges from 0 to 6.", "Set Belly Burn Damage.", host.vore_selected.digest_burn, 6, 0, round_value=FALSE)
+			// var/new_damage = tgui_input_number(user, "Choose the amount of burn damage prey will take per tick. Ranges from 0 to 6.", "Set Belly Burn Damage.", host.vore_selected.digest_burn, 6, 0, round_value=FALSE)
+			// CHOMP Edit
+			var/old_damage = host.vore_selected.digest_burn
+			var/max_total_damage = host.vore_selected.digest_max
+			var/current_total_damage = host.vore_selected.get_total_digestion_damage()
+			var/new_damage = tgui_input_number(user, "Choose the amount of burn damage prey will take per tick. Max of 36 across all damage types.", "Set Belly Burn Damage.", old_damage, max_total_damage, 0, round_value=FALSE)
```

Comment:
No, those shouldn't use the max, those should be calculated dynamically in this case.

Reply:
Redid those entirely to be a lot cleaner.